docs/view_components/flow_chart.rst

```diff
@@ -1,11 +1,34 @@
-Component title
----------------
+.. _flowchart-label:
+
+Flow-chart
+----------
+The flow chart is a map of all the sequential steps of the simulation in one view-component. The below sections present the proposed structure for this view-component.
+
+The following are the advantages of the flow chart view-component:
+
+* Helps in sequentially progressing through the modeling and simulation process
+* Allows the user to jump between different steps of the simulation
+* Brings a design consistency and uniformity to the UI of the tool
 
 Attributes
 ^^^^^^^^^^
 
+* Clickable, identical buttons
+* Ribbon shaped, clickable icons/buttons placed sequentially as per the simulation progression order.
+* Each button is a step in the simulation process
+* Initially, only the first step (or button) is highlighted in a certain color, which notifies the user that the view associated with that step has to be fulfilled (provided with values for the parameters in that view-component) before they can progress to further steps.
+* Rest of the buttons are greyed out.
+* All the buttons for which the requirements of the associated simulation steps have been fulfilled turn green (or a different color).
```

Comment:
```suggestion
* Each step has to be completed one at a time. Once a step has met its minimum requirements, the button of the next step becomes clickable. Therefore, the user can come back to the steps he has already seen but he cannot click a random step. 
```

Reply:
This is not an attribute but a requirement